modApiServer/src/org/aion/api/server/http/undertow/UndertowRpcServer.java

```diff
@@ -26,16 +16,17 @@
 import java.util.Map;
 
 public class UndertowRpcServer extends RpcServer {
+
```

Comment:
The `RpcServer` class could include more of the common functionality between `UndertowRpcServer` and `NanoRpcServer`.

Reply:
I don't think there are any more commonalities that could be easily pulled up to the base class without expending some effort generifying and refactoring, which would unneccessarily complicate the implementation. 

If you had some specific ideas, I'm all ears :) (but I think the implementations are simple enough and don't thing we have alot to gain by refactoring this class heirarchy).

Reply:
I was thinking that the `LOG` and `CORS_HEADERS` can be placed in the abstract class and perhaps the log debug printout with the options display.